docs/getting-started/step1.md

```diff
@@ -41,7 +41,7 @@ sidebarDepth: 2
 3. 点击 **Artifacts** 展开下拉菜单, 然后根据你的平台选择一个二进制包并下载。
 
 :::tip 提示
-你必须先登录 GitHub 账号才能白嫖 GitHub Actions 中的 Artifacts~
+你必须先登录 GitHub 账号才能白嫖 GitHub Actions 中的 Artifacts~，或者去[店小二](https://t.me/Qvreleasebot)那边下载 ~~（虽然这个也需要登录 Telegram 账号）~~
```

Comment:
```suggestion
你必须先登录 GitHub 账号才能白嫖 GitHub Actions 中的 Artifacts。如您持有 Telegram 账户，也可使用 [Qv2ray 店小二机器人](https://t.me/Qvreleasebot) 以下载最新版本的 Actions Artifact。
```

Reply:
> It's totally fine we don't make an English version, since the bot is in Chinese.
> Resolve the conversation before merging. Thank you.

But if there is a plan for non-Chinese users, it is recommended to design an English version for the robot